### ISSUE TITLE
Call log modal focus order differs from visual order

### ISSUE THREAD

**Issue by henrycatalinismith** [henrycatalinismith/zetkin#8]
## Steps to reproduce

1. Open an assignment
2. Click the log button
3. Start your screen reader
4. Focus the close button

## Expected outcome

The close button is visually first so it should be first in the focus order. See [C27: Making the DOM order match the visual order](https://www.w3.org/TR/WCAG20-TECHS/C27.html) for details of the success criteria this addresses and the assistive technology use cases it supports.

## Actual outcome

The close button is at the end of the focus order despite being at the top left of the screen.

![VoiceOver focusing the close button at the end of the focus order](https://user-images.githubusercontent.com/566159/191597491-40c0faf1-2036-44ba-934b-31a9c71df5a8.gif)

## Recommendation

Rearrange the DOM order so that `.OverlayStack-closeButton` is before `.OverlayStack-overlay`.
